eng/send-to-helix-step.yml

```diff
@@ -39,9 +45,20 @@ steps:
       _TimeoutPerTestInMinutes: ${{ parameters.timeoutPerTestInMinutes }}
       ${{ if eq(parameters.publishTestResults, 'true') }}:
         SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+      # TODO: remove NUGET_PACKAGES once https://github.com/dotnet/arcade/issues/1578 is fixed
+      NUGET_PACKAGES: $(Build.SourcesDirectory)\.packages 
 
 - ${{ if ne(parameters.osGroup, 'Windows_NT') }}:
-  - script: ./eng/common/msbuild.sh --ci tests/helixpublishwitharcade.proj /maxcpucount
+  - script: ./eng/common/build.sh /p:DotNetPublishToBlobFeed=true --ci --restore --projects $(Build.SourcesDirectory)/eng/empty.proj
+    displayName: Restore blob feed tasks
+    ${{ if ne(parameters.condition, '') }}:
+      condition: ${{ parameters.condition }}
+    ${{ if eq(parameters.osGroup, 'FreeBSD') }}:
+      env:
+        # Arcade uses this SDK instead of trying to restore one.
+        DotNetCoreSdkDir: /usr/local/dotnet
+
+  - script: ./eng/common/msbuild.sh --ci ${{ parameters.helixProjectFile }} /maxcpucount /bl:$(Build.SourcesDirectory)/bin/Logs/SendToHelix.binlog
```

Comment:
Thanks for adding binlogging to the step. I had this on my backlog for a while..

I like that the coreclr Helix submission projects (we have two now) have the same entrypoint via send-to-helix-step.yml and presumably the same (similar) API (i.e. a set of MSBuild properties specified here)

Reply:
Yes, the set of msbuild properties are the same.  I would have combined, but the way coreclr tests are run is way to different from the way I want to run the corefx tests.

I guess this is a similar argument for also having a separate test-job.yml for corefx tests.